blockbase/block-template-parts/header.html

```diff
@@ -1,6 +1,6 @@
 <!-- wp:group {"align":"full","tagName":"header","style":{"spacing":{"padding":{"right":"35px","left":"35px"}}},"className":"site-header"} -->
 <header class="wp-block-group alignfull site-header" style="padding-right:35px;padding-left:35px">
```

Comment:
While we are here, I noticed that we have currently have header inception because we are rendering a template part with a header tag and then a group inside of it with a header tag. We should consolidate?

<img width="725" alt="Screen Shot 2021-05-27 at 4 04 35 PM" src="https://user-images.githubusercontent.com/5375500/119890002-22c7a280-beec-11eb-8e27-b8735681e525.png">

Reply:
I think I fixed that same thing in Quadrat too.  Yes we should.  I don't believe the element in the template part should be a `<header>` but just a `<div>`.